### ISSUE TITLE
Insert a new step after Recurrence' button has focusable descendants

### ISSUE THREAD

**Issue by Xiaoyu-Huang** [Azure/LogicAppsUX#5274]
### Describe the issue 

1\.Navigate to add trigger and search for 'Schedule'. Add 'Recurrence' option.

2. Run the AI tool and observe whether 'Insert a new step after Recurrence' button has focusable descendants or not. 

Expected Result:

 'Insert a new step after Recurrence' button should not have focusable descendants or unnecessary role= "button" should be removed from the

<div>. <p></p><h3 id="are-you-using-new-designer-or-old-designer">Are you using new designer or old designer </h3><p>New Designer (With action panels) </p><h3 id="screenshots-or-videos">Screenshots or Videos </h3><p><img src="https://github.com/user-attachments/assets/1ab615cd-cfe6-472e-a0c0-e7fe0fb7607d" alt="image">¨NBSP;</p><p>Verfied in Azure portal: same issue could be reproed </p><p><img src="https://github.com/user-attachments/assets/77e927bc-4356-4281-a18e-1ccaba48ccc8" alt="image"> </p><h3 id="browser">Browser </h3><p>Edge </p><h3 id="additional-context">Additional context </h3><p><em>No response</em> </p></div>

**Comment by hartra344** [comment]
This is fine in this case. We do the work to make sure screen readers and focus is handled correctly. There's no way to build these UI elements to work/layout correctly otherwise though.
